lib/full-site-editing/templates.php

```diff
@@ -188,10 +188,16 @@ function gutenberg_the_skip_link() {
 		return;
 	}
 
-	// Early exit if not an FSE theme.
+	// Early exit if not a block theme.
 	if ( ! gutenberg_supports_block_templates() ) {
 		return;
 	}
+
+	// Early exit if not a block template.
+	global $_wp_current_template_content;
+	if ( ! $_wp_current_template_content ) {
```

Comment:
how confident are you about the future-proof aspect of that check? Is there a way we can add a test for this?

Reply:
Well... Ideally we'd have an `is_block_template()` function or something similar, but since we don't have something like that, checking for `$_wp_current_template_content` seems like the safest option.
That global var is only set in https://github.com/WordPress/WordPress/blob/4ae0e4220f0c01005afa0dcbb74979c0d5cb9abc/wp-includes/block-template.php#L45-L58 in WordPress Core, and https://github.com/WordPress/gutenberg/blob/c746132fff812a9106cdd75cd36a017e762bb455/lib/full-site-editing/template-loader.php#L61-L74 in Gutenberg.
So in both cases it's only not-null when we're in a block template.

Even if we had a function `is_block_template()`, I can't think of any other way to do it except checking for that var :thinking: